server/src/main/java/org/opensearch/common/util/FeatureFlags.java

```diff
@@ -64,6 +64,11 @@ public class FeatureFlags {
      */
     public static final String DOC_ID_FUZZY_SET = "opensearch.experimental.optimize_doc_id_lookup.fuzzy_set.enabled";
 
+    /**
+     * Gates the functionality of tiered caching
+     */
+    public static final String TIERED_CACHING = "opensearch.experimental.feature.tiered.caching.enabled";
+
```

Comment:
Can you please add this to https://github.com/opensearch-project/OpenSearch/blob/main/distribution/src/config/opensearch.yml with a comment that explains the feature in a single sentence?

Ultimately, the feature flag is controlling pluggable caching, right? Tiered caching is one implementation.